mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.java

```diff
@@ -1069,6 +1069,8 @@ public void bind(Widget widget) {
                     mConnection.getUsername(), mConnection.getPassword()));
             mWebView.getSettings().setDomStorageEnabled(true);
             mWebView.getSettings().setJavaScriptEnabled(true);
+            mWebView.getSettings().setSupportMultipleWindows(true);
```

Comment:
I guess we should move the duplicate (here and in WebViewFragment) call to a helper function. Maybe to `Util.applyAuthentication()` and rename it to `Util.initWebview()`.

> If set to true, WebChromeClient#onCreateWindow must be implemented by the host application. The default is false.

https://developer.android.com/reference/android/webkit/WebSettings.html#setSupportMultipleWindows(boolean)

Reply:
And if you look at ChromeWebClient, the default implementation of onCreateWindow() always returns false, so those links get blocked.